main/guides/orchestration/getting-started/contract-walkthrough/send-anywhere.md

```diff
@@ -0,0 +1,207 @@
+# SendAnywhere Contract Walkthrough
+
+The "Send Anywhere" contract is designed to facilitate the transfer of assets from one chain to another using
+Agoric's [Orchestration](/glossary/#orchestration) library. The contract allows a user to send assets of a specific brand
+to a destination address on any supported blockchain.
+
+The high-level flow of the contract is:
+
+- Validates the asset brand on the source chain.
+- Creates or ensures the existence of a local account (held by the contract) to handle temporary holding.
+- Transfers the asset from the source address to the local account.
+- Initiates a transfer to the destination address, which could reside on the remote blockchain.
+- Gracefully handles the failures.
+
+The contract is implemented in two separate files:
+
+1. `send-anywhere.contract.js` implements the `start` function of the contract to initialize the contract and
+   expose `publicFacet` and `creatorFacet`.
```

Comment:
There's only this mention about this contract's `creatorFacet` and nothing else on the page. We should add a line or two about it.

Reply:
Removing mention of `creatorFacet` altogether for now.